scipy/stats/_relative_risk.py

```diff
@@ -0,0 +1,266 @@
+
+import operator
+from collections import namedtuple
+from dataclasses import dataclass
+import numpy as np
+from scipy.special import ndtri
+
+
+def _validate_int(n, bound, name):
+    msg = f'{name} must be an integer not less than {bound}, but got {n!r}'
+    try:
+        n = operator.index(n)
+    except TypeError:
+        raise TypeError(msg) from None
+    if n < bound:
+        raise ValueError(msg)
+    return n
+
+
+@dataclass
+class ConfidenceInterval:
+    low : float
+    high : float
+
+
+class RelativeRiskResult:
+    """
+    Result of scipy.stats.relative_risk.
+
+    Attributes
+    ----------
+    exposed_cases : int
+        The number of "cases" (i.e. occurrence of disease or other event
+        of interest) among the sample of "exposed" individuals.
+    exposed_total : int
+        The total number of "exposed" individuals in the sample.
+    control_cases : int
+        The number of "cases" among the sample of "control" or non-exposed
+        individuals.
+    control_total : int
+        The total number of "control" individuals in the sample.
+    relative_risk : float
+        This is::
+
+            (exposed_cases/exposed_total) / (control_cases/control_total)
+
+    Methods
+    -------
+    confidence_interval(confidence_level=0.95)
+        Compute the confidence interval of the relative risk for a given
+        confidence level.
+
+    """
+    def __init__(self, exposed_cases, exposed_total,
+                 control_cases, control_total,
+                 relative_risk):
+        self.relative_risk = relative_risk
+        self.exposed_cases = exposed_cases
+        self.exposed_total = exposed_total
+        self.control_cases = control_cases
+        self.control_total = control_total
+
+    def __repr__(self):
+        return f"RelativeRiskResult(relative_risk={self.relative_risk})"
+
+    def confidence_interval(self, confidence_level=0.95):
+        """
+        Compute the confidence interval for the relative risk.
+
+        The confidence interval is computed using the Katz method
```

Comment:
There are three methods in Katz' paper. Maybe mention which this is? C, I think?

Reply:
Done.